<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StrMonDialog</class>
 <widget class="QWidget" name="StrMonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input Stream Monitor</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="wgPanel1" native="true">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text" stdset="0">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="cBSelectFormat">
        <property name="toolTip">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>Stream</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HEX</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ASCII</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="toolTip">
         <string>Clear</string>
        </property>
        <property name="icon">
         <iconset resource="strsvr_qt.qrc">
          <normaloff>:/buttons/del.bmp</normaloff>:/buttons/del.bmp</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="toolTip">
         <string>Pause</string>
        </property>
        <property name="icon">
         <iconset resource="strsvr_qt.qrc">
          <normaloff>:/buttons/pause.bmp</normaloff>:/buttons/pause.bmp</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDown">
        <property name="toolTip">
         <string>Scroll Down</string>
        </property>
        <property name="icon">
         <iconset resource="strsvr_qt.qrc">
          <normaloff>:/buttons/fitd.bmp</normaloff>:/buttons/fitd.bmp</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tWConsole"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="strsvr_qt.qrc"/>
 </resources>
 <connections/>
</ui>
